wp-content/plugins/tw-import-post-types/tw-import-post-types.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Plugin Name: TW Import Post Types
+ * Description: Creates the Custom Post Types necessary to import content from the old site. Can be REMOVED after the import is complete.
+ *
+ * @package tw_import_post_types
+ */
+
+ function cptui_register_my_cpts() {
+
+	/**
+	 * Post Type: Persons.
+	 */
+
+	$labels = [
+		"name" => esc_html__( "Persons", "newspack" ),
+		"singular_name" => esc_html__( "Person", "newspack" ),
+		"all_items" => esc_html__( "Persons", "newspack" ),
+		"add_new" => esc_html__( "Add New", "newspack" ),
+		"add_new_item" => esc_html__( "Add New Person", "newspack" ),
+		"edit_item" => esc_html__( "Edit Person", "newspack" ),
+		"new_item" => esc_html__( "New Person", "newspack" ),
+		"view_item" => esc_html__( "View Person", "newspack" ),
+		"search_items" => esc_html__( "Search Person", "newspack" ),
+		"not_found" => esc_html__( "No Person found", "newspack" ),
+		"not_found_in_trash" => esc_html__( "No Person found in Trash", "newspack" ),
+	];
+
+	$args = [
+		"label" => esc_html__( "Persons", "newspack" ),
+		"labels" => $labels,
+		"description" => "Use person to create a page for each person associated with a story, episode, etc.",
+		"public" => true,
+		"publicly_queryable" => true,
+		"show_ui" => true,
+		"show_in_rest" => false,
+		"rest_base" => "",
+		"rest_controller_class" => "WP_REST_Posts_Controller",
+		"rest_namespace" => "wp/v2",
+		"has_archive" => true,
+		"show_in_menu" => true,
+		"show_in_nav_menus" => true,
+		"delete_with_user" => false,
+		"exclude_from_search" => false,
+		"capability_type" => "post",
+		"map_meta_cap" => true,
+		"hierarchical" => false,
+		"can_export" => true,
+		"rewrite" => [ "slug" => "person", "with_front" => true ],
```

Comment:
Can we change these slugs to avoid collision with the taxonomies' slugs?

Reply:
Yes we can, i just made a copy/paste from what we had in CPT UI.

Reply:
Done @rpeterman-gp , i added a "-post" as suffix